src/lib/operators/join_helper/join_output_writing.hpp

```diff
@@ -183,12 +182,13 @@ inline std::vector<std::shared_ptr<Chunk>> write_output_chunks(
   PosListsByChunk left_side_pos_lists_by_segment;
   PosListsByChunk right_side_pos_lists_by_segment;
 
-  // left_side_pos_lists_by_segment will only be needed if build is a reference table and being output
+  // left_side_pos_lists_by_segment will only be needed if the left input table is a reference table and being an
```

Comment:
I don't get "and being an output table."

Reply:
Sorry, but I didn't write the code. So I'm not really sure about a more precise comment.

Reply:
Before this PR, this was

> build_side_pos_lists_by_segment will only be needed if build is a reference table and being output

Knowing that the build side might not be part of the output and reading "output" not as a noun but as a participle adjective (read: an [*outputted*](https://english.stackexchange.com/questions/35418/past-tense-of-to-output-output-or-outputted) table)

Honestly, I think this comment is pointless right now.

Reply:
ok, then I will remove it.